charts/nucliadb/values.yaml

```diff
@@ -15,6 +15,9 @@ env:
   cluster_discovery_mode: kubernetes
   cluster_discovery_kubernetes_namespace: nucliadb
   cluster_discovery_kubernetes_selector: "app.kubernetes.io/name=node"
+  # when we're k8s, we want structured logs to stdout
```

Comment:
can you add the reason in the comment? "so that they are directly scraped to gcloud logging system" or something alike